up42/viztools.py

```diff
@@ -47,21 +49,21 @@ def __init__(self):
     def plot_results(
         self,
         figsize: Tuple[int, int] = (14, 8),
-        filepaths: Union[List[Union[str, Path]], Dict] = None,
+        bands: List[int] = [1, 2, 3],
```

Comment:
Using a Tuple here instead of a list might enable you to remove the pylint disable.

Reply:
I the user wants to give a single band, would need to be explicitly a tuple e.g. with additional comma `bands=(1,)`, probably leads to problems.